core/base/src/TSystem.cxx

```diff
@@ -3353,6 +3357,33 @@ int TSystem::CompileMacro(const char *filename, Option_t *opt,
       linkDepLibraries = linkLibs & 0x1;
    }
 
+   bool useCxxModules = false;
```

Comment:
Isn't that a `constexpr const bool`? Or is the code changing that after line 3362? Maybe
```
constexpr const bool useCxxModules =
#ifdef R__USE_CXXMODULES
    true;
#else
    false;
#endif
```